modules/lang-painless/src/main/java/org/elasticsearch/painless/toxcontent/DecorationToXContent.java

```diff
@@ -291,8 +291,6 @@ public static void ToXContent(ReferenceDecoration referenceDecoration, XContentB
             builder.endArray();
         }
 
-        builder.field("factoryMethodType");
```

Comment:
Do folk no longer need this?

Reply:
It's private now, so inaccessible.  I'll bet I can provide a replacement getter that still provides useful information.

Reply:
Could you explain what it's for? If no one's using it then I'd fine pitching it. If folks were using it they probably want to know. If it was just debugging stuff then cool, add whatever makes you feel good.